tests/gatekeeper.test.ts

```diff
@@ -3100,10 +3169,15 @@ describe('isValidDID', () => {
     });
 
     it('should return false for wrong type', async () => {
+        // @ts-expect-error Testing private state
```

Comment:
I'm confused. We are not testing private state here, are we?

Reply:
Copy and paste from the first usage, will update subsequent usages to something more meaningful.